src/battle-dex.ts

```diff
@@ -528,6 +528,12 @@ const Dex = new class implements ModdedDex {
 			return spriteData;
 		}
 
+		// Digimon Crys
+		if (options.digi) {
```

Comment:
Why not instead make this generic, like `options.mod` and make it a `mod?: string`, and set it to `'digimon'` for digi. Then below, do `sprites/${options.mod}/audio` instead, so another mod might be able to use the same code.